client/src/Login.js

```diff
@@ -28,7 +30,7 @@ class Login extends React.Component {
       })
       .catch(err => {
         if (err.response.status === 409) {
-          window.alert('You have already created an account with this email! Please check your email to login.')
+          return this.setState({ status: 1 })
```

Comment:
[nit] Rather than using an arbitrary number, you can consider something more descriptive like `alertType: duplicateAccount`